### DISCUSSION TITLE
en/where-hcu-did-not-do-as-much-core-update-did/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#97] [General]
# en/where-hcu-did-not-do-as-much-core-update-did/

Free world, free thoughts, free words.

https://dariusz.wieckiewicz.org/en/where-hcu-did-not-do-as-much-core-update-did/

**Comment by danielfdickinson** [comment]
Hi. I jwanted to thank you for this post. I try to stick to the content (while still looking good) myself and it is good to see that being the best SEO practice. I even mentioned that in the slides for a presentation about website development and design for local folks (I'm not putting the link here as my goal is not to spam your comments section, but to express appreciation, but if you want it, I'm happy to pass it on). I had noticed that my stats for my sites have been steadily increasing without doing anything special (I do take some care to have proper metadata and seek to be mobile friendly and accessible, which I view more as baseline technical hygiene than SEO as such).
In short, thank you for this and the morale boost for those who avoid shady SEO practices.

**Comment by Donaldx9** [comment]
I like this blog more and more.
